CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ cmake_minimum_required(VERSION 3.15)
 # Get the library version
 include(cmake/oidn_version.cmake)
 
+# Load the dynamically generated sysroot configuration
+include(sysroot_config.cmake)
+
```

Comment:
This should be removed. The sysroot is automatically set by CMake when building for iOS with `-DCMAKE_SYSTEM_NAME=iOS`.